.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,41 @@
+name: PreRelease
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  Publish:
+    name: PreRelease
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.ADMIN_TOKEN }}
+          ref: ${{ github.ref }}
+          # pulls all commits (needed for lerna / semantic release to correctly version)
+          fetch-depth: '0'
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+
+      - name: Install Packages
+        run: yarn install
+
+      - name: Build
+        run: yarn build
+
+      - name: Authenticate with Registry
+        run: |
+          yarn logout
+          echo "registry=http://registry.npmjs.org/" >> .npmrc
+          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
+          npm whoami
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Publish package
+        run: |
+          yarn publish:ci:prerelease --preid $(echo ${GITHUB_REF##*/}) patch
```

Comment:
I think you can also take the branch name from `github.ref`

Reply:
`github.ref` = 'refs/heads/master'
`GITHUB_REF##*/` = 'master'